pyogrio/tests/test_core.py

```diff
@@ -451,6 +451,7 @@ def test_read_info_force_total_bounds(
         assert info["total_bounds"] is None
 
 
+@pytest.mark.filterwarnings("ignore: Measured")
```

Comment:
I'm not observing this warning locally when running tests; are you still seeing it?

Reply:
No indeed... removed.